addons/payment_adyen/models/payment_transaction.py

```diff
@@ -93,13 +93,17 @@ def _send_payment_request(self):
         if not self.provider_id.capture_manually:
             data.update(captureDelayHours=0)
 
-        # Make the payment request to Adyen
-        response_content = self.provider_id._adyen_make_request(
-            url_field_name='adyen_checkout_api_url',
-            endpoint='/payments',
-            payload=data,
-            method='POST',
-        )
+        try:
+            # Make the payment request to Adyen
```

Comment:
Nitpicking but I feel like the comment concerns the whole block.
```suggestion
        # Make the payment request to Adyen
        try:
```